libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -472,46 +465,7 @@ private void updateProgressChangeListener() {
 
   @Nullable
   private MapRouteProgressChangeListener buildMapRouteProgressChangeListener() {
-    shutdownVanishingRouteLineAnimator();
-    if (vanishRouteLineEnabled) {
-      vanishingRouteLineAnimator = new RouteLineValueAnimator();
-    } else {
-      vanishingRouteLineAnimator = null;
-    }
-
-    float vanishingPoint = (this.mapRouteProgressChangeListener != null)
-            ? this.mapRouteProgressChangeListener.getPercentDistanceTraveled() : 0f;
-    MapRouteProgressChangeListener newListener = new MapRouteProgressChangeListener(
-            this.routeLine,
-            routeArrow,
-            vanishingRouteLineAnimator
-    );
-    newListener.updatePercentDistanceTraveled(vanishingPoint);
-    return newListener;
-  }
-
-  /**
-   * Returns the percentage of the distance traveled that was last calculated. This is only
-   * calculated if the vanishing route line feature is enabled.
-   *
-   * @return the value calculated during the last progress update event or 0 if not enabled.
-   */
-  public float getPercentDistanceTraveled() {
```

Comment:
This is breaking SemVer

Reply:
We're going to keep this change and remove this in the 1.0 branch per our discussion.